boards/shields/adafruit_featherwing_128x32_OLED/shield.yml

```
@@ -0,0 +1,7 @@
+shields:
+  - name: adafruit_featherwing_128x32_OLED
```

all lower-case please -- fix everywhere (rename folder, overlay, picture file, etc.)

```suggestion
  - name: adafruit_featherwing_128x32_oled
```